kubetest/util.go

```diff
@@ -140,8 +141,14 @@ func finishRunning(cmd *exec.Cmd) error {
 
 	finished := make(chan error)
 
+	sig_channel := make(chan os.Signal, 1)
+	signal.Notify(sig_channel, os.Interrupt)
+
 	go func() {
 		finished <- cmd.Wait()
+		sig := <-sig_channel
+		log.Printf("Killing %v after receiving %s", -cmd.Process.Pid, sig)
+		syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL)
```

Comment:
`fmt.Errorf()` instantiates an error object for you to return, which I agree is not what we want to do here. However this does not preclude us from logging the error with a `log.Printf("blah blah: %v", err)` and then continuing